### PR TITLE
[Component with its own class name as service type] LPS-193512 Refactor Cache classes from portal-template module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-193512

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/portal-template/portal-template-freemarker/src/main/java/com/liferay/portal/template/freemarker/internal/FreeMarkerManager.java [hunk 1]
```diff
@@ -324,7 +392,9 @@ protected TemplateContextHelper getTemplateContextHelper() {
 	}
 
 	@Modified
-	protected void modified(ComponentContext componentContext) {
+	protected void modified(
+		ComponentContext componentContext, BundleContext bundleContext) {
```
Comment (rafaprax) [inline modules/apps/portal-template/portal-template-freemarker/src/main/java/com/liferay/portal/template/freemarker/internal/FreeMarkerManager.java:396]: sort parameters

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Hey @MrJohn1911, let's discuss this a little bit more this solution today.

**Pull request closed without merging**